lib/telephony/otp_sender.rb

```diff
@@ -96,7 +96,9 @@ def log_response(response, context:)
     def otp_transformed_for_channel
       return otp if channel != :voice
 
-      otp.chars.join(" <break time='#{Telephony.config.voice_pause_time}' /> ")
+      break_time = " <break time='#{Telephony.config.voice_pause_time}' /> "
+      transformed_otp = otp.chars.join(break_time)
+      transformed_otp + break_time
```

Comment:
another trick is you can add an empty string to get an extra instance in there
```suggestion
      [*otp.chars, ''].join(" <break time='#{Telephony.config.voice_pause_time}' /> ")
```
same result of adding an extra bit at the end